Generating Guitar Model & Controller With Empty CRUD Actions + Presenting Data from the Actual DB

LiorKGOW opened the pull request:
- Generate Guitars controller with empty CRUD actions using rails generate controller
- Generate Guitar model with the first migration using rails generate model
- Adding resourceful routes to guitars controller
- Presenting data from the DB instead of some mock data & Adding some seed to populate the DB with a few guitars

Screenshot using the Guitars found in the seed.rb file: 
(when you enter the localhost on the browser after running the server the application should look like this)
![image](https://user-images.githubusercontent.com/93318917/210735642-1ca811fd-321d-42d6-8272-82d8f10eb46c.png)

MariaAga commented on test/models/guitar_test.rb:
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class GuitarTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Please add a test or delete the file

LiorKGOW replied:
File removed 👍🏼 
Resolving this conversation

MariaAga commented on test/fixtures/guitars.yml:
```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Can be deleted

LiorKGOW replied:
Will do 👍🏼

MariaAga commented on app/views/guitars/delete.html.erb:
```diff
@@ -0,0 +1 @@
+<h1>Guitars#delete</h1>
```
Since we will be using the CRUD actions as api only, the template files for them can be deleted

LiorKGOW replied:
OK I'll remove them 👍🏼

MariaAga reviewed (changes requested):
Tested and works, some auto generated code/files can be removed as in the comments:

LiorKGOW commented:
@MariaAga @Ron-Lavi 
I have merged this PR with the content of #33 
In addition, I have changed the commits to be more precise and small
Please when you review have a look at each commit individually 🙏🏼  hopefully they would make better sense separately
You can see the top comment of this PR with the descriptions of the commits
Also, added a screenshot after populating the DB with the seed using `bundle exec rake db:seed`, you should see a new guitar when you enter the localhost after running the server of course :)

LiorKGOW commented:
> Tested and works, some auto generated code/files can be removed as in the comments:

Thank you Maria for the review
I have included the fixes 👍🏼

MariaAga commented on config/routes.rb:
```diff
@@ -3,5 +3,12 @@
+
+  resources :guitars do
+    member do
+      get :delete
```
Why only delete?

LiorKGOW replied:
Removed 👍🏼

MariaAga reviewed (approved):
tested and went over the code, looks good!

LiorKGOW commented:
> tested and went over the code, looks good!

Thank you @MariaAga  :)
